src/deploy/remoteconfig/index.ts

```diff
@@ -0,0 +1,6 @@
+"use strict";
```

Comment:
`use strict` isn't necessary in TS files (applies throughout)

(wondering if there's a lint rule for this... I'll have to look)